MPCHomeControl/src/tools/mod.rs

```diff
@@ -0,0 +1,34 @@
+extern crate nalgebra as na;
+
+use chrono::prelude::*;
+use na::{Dot, Norm, Vector3};
+use spa;
+
+pub fn calc_solar_vector(azimuth: f64, zenith_angle: f64) -> Vector3<f64> {
```

Comment:
Ocekavas ze budes potrebovat pocitat solar vektor bez volani `spa` ? Protoze jinak bych to zabalil aby ta funkce mela rovnou parametr lat/lon/time

Reply:
V jakem reference frame je vraceny vektor? To vypada jako nejaky lokalni coordinate system vazany na svetove strany, ze?Muj prvni reflex bylo pouzivat [ECEF](https://en.wikipedia.org/wiki/Earth-centered,_Earth-fixed_coordinate_system), ale to je asi overkill. Ale nekam to tam napis at pak nejsme (=nejsem) zmateni.

Reply:
jj melo by to byt vazany k severu

Reply:
to musim jeste vyresit tohle. je to prvni nastrel co jsem vcera nakodil v aute